tests/get-coverage.yml

```diff
@@ -1,19 +1,22 @@
 # SPDX-License-Identifier: BSD-3-Clause
 ---
+# This expects the variable test_playbook to be set from the outside
 - name: Prepare for coverage extraction
   hosts: all
   tasks:
+    # Use set_fact to set variables to make them available in all playbooks
+    # vars: Would only set variables for the current playbook
```

Comment:
"all plays" and "current play" is probably more accurate, as your example shows.

Reply:
fixed